configs/catalog-entities/users.override.example.yaml

```
@@ -0,0 +1,10 @@
+# This is an example override file for catalog users.
+# If present, this file will be used instead of users.yaml.
+# Uncomment and edit the entries below to override catalog users.
+
+apiVersion: backstage.io/v1alpha1
+kind: User
+metadata:
+  name: user
+spec:
+  memberOf: [default]
```

```suggestion
  name: my-user
spec:
  profile:
    displayName: My User
  memberOf:
    - my-group1
    - my-group2
```

Suggestion to use something different than the default as an example.